content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+To simplify using function-level test reporting, O3DE provides wrapper code to register tests for frameworks such as GoogleTest and PyTest into CTest. These wrappers enable writing tests which run on any operating system. XML files are generated to track the low-level results not reported in CTest, as well as artifacts such as log output and crash dumps.
+
+In short, all tests in O3DE use CTest as a high level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate to this directory:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration>
+```
+
+You may not want to run the commands above without also filtering the set of tests to run. Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL + C` . The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following ctest command filters to only tests in these two suites:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
+```
+
+The build configuration should be one you have already built locally. Running the Smoke and Main suites for the `profile` build configuration on a Linux machine should look like:
+
+```shell
+cd user/github/o3de/build/linux
+ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+```
+
+After running CTest, results save to `.../<build_folder>/Testing/`.  If you prefer to see a summary of failures in your terminal, add the flag `--output-on-failure`.
+
+For more information on CTest usage, refer to its [online documentation](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
+
+### Adding test modules to CTest
+
+CTest registers entire modules of test code. These modules typically contain multiple individual tests of the same feature. Adding new tests has three major steps, with a prerequisite:
```

Comment:
It may be worth pointing out that if you're simply writing new tests, most of these steps do not need to be done.

Reply:
"Writing new tests" is a pretty vague motivation, though I think this statement would apply to "expanding existing tests"

The following suggestions do suggest where to create/look-for this file, such as Line 127, so I didn't think it necessary to add here at the top.